tests/Feature/Connect/LeaderboardInfoTest.php

```diff
@@ -0,0 +1,123 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Tests\Feature\Connect;
+
+use App\Models\Game;
+use App\Models\Leaderboard;
+use App\Models\System;
+use App\Models\User;
+use App\Platform\Enums\ValueFormat;
+use Illuminate\Foundation\Testing\RefreshDatabase;
+use Illuminate\Support\Carbon;
+use Tests\TestCase;
+
+class LeaderboardInfoTest extends TestCase
+{
+    use BootstrapsConnect;
+    use RefreshDatabase;
+
+    public function testLeaderboardInfo(): void
+    {
+        $now = Carbon::now();
+        Carbon::setTestNow($now);
+
+        /** @var System $system */
+        $system = System::factory()->create();
+        /** @var Game $game */
+        $game = Game::factory()->create(['ConsoleID' => $system->id]);
+        /** @var Leaderboard $leaderboard */
+        $leaderboard = Leaderboard::factory()->create(['GameID' => $game->id]);
+
+        $this->get($this->apiUrl('lbinfo', ['i' => $leaderboard->id]))
+            ->assertStatus(200)
+            ->assertExactJson([
+                'Success' => true,
+                'LeaderboardData' => [
+                    'Entries' => [],
+                    'GameID' => $game->id,
+                    'LBAuthor' => $leaderboard->developer?->User,
+                    'LBCreated' => $now->format('Y-m-d H:i:s'),
+                    'LBDesc' => $leaderboard->description,
+                    'LBFormat' => $leaderboard->format,
+                    'LBID' => $leaderboard->id,
+                    'LBMem' => $leaderboard->Mem,
+                    'LBTitle' => $leaderboard->title,
+                    'LBUpdated' => $now->format('Y-m-d H:i:s'),
+                    'LowerIsBetter' => (int) $leaderboard->rank_asc,
+                    'TotalEntries' => 0,
+                ],
+            ]);
+    }
+
+    public function testLeaderboardInfoWithEntries(): void
+    {
+        $now = Carbon::now();
+        Carbon::setTestNow($now);
+
+        /** @var System $system */
+        $system = System::factory()->create();
+        /** @var Game $game */
+        $game = Game::factory()->create(['ConsoleID' => $system->id]);
+        /** @var Leaderboard $leaderboard */
+        $leaderboard = Leaderboard::factory()->create([
+            'GameID' => $game->id,
+            'LowerIsBetter' => false,
+            'Format' => ValueFormat::Score,
+        ]);
```

Comment:
Suggest continuing the first test here to avoid rebuilding the database between tests. Start with the empty leaderboard fetch, add some entries, then fetch it again.

Reply:
Done in latest. I am hoping when we upgrade to Laravel 11, we won't need to do this anymore ([ref](https://laravel.com/docs/11.x/releases#database-performance)).

To better separate the two test cases, I have included `#region` markers so they can at least be folded.

![Screenshot 2024-07-17 at 8 27 32 PM](https://github.com/user-attachments/assets/be448253-7668-4a8f-a5b3-3bcfcb34bdcb)